_posts/0000-01-02-milangeorge98.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
### Hello World !
Hope you are having a great day !
Cheers
And 
Happy coding !